bin/supporters-mongo-1-to-2.js

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env nodejs
+var _ = require('lodash');
+var Promise = require('bluebird');
+var debug = require('debug')('supporters-1-to-2');
+var nconf = require('nconf');
+
+var mongo = require('../lib/mongo');
+
+var cfgFile = "config/settings.json";
+nconf.argv().env().file({ file: cfgFile });
+
+
+function takeTheGood(memo, user) {
+
+    if(_.isUndefined(user.publicKey))
+        return memo;
+
+    var good = {
+        publicKey:  user.publicKey,
+        keyTime: new Date(user.keyTime),
+        lastInfo: new Date(user.lastInfo),
```

Comment:
what is `lastInfo`? if it's the last timestamp the user was active, what do you think about `lastAction` or `lastActivity`?